src/pages/guides/index.md

```diff
@@ -33,9 +35,13 @@ So you've just created a new template for App Builder and want to make it availa
 
 ### install.yml
 
-- The `categories` field in `install.yml` must not be empty. Template must have categories defined.
-- If the template implements an extension point, `install.yml` should define the `extension` field. It must contain the extension point metadata in the `serviceCode` property. `extension` is an optional property.
-- If the template contains the `services` property to declare the services it uses, `services` must be an array of objects representing a service the template depends on. Each object must contain the `code` property. `services` is an optional property.
+| Field        | Type | Description                                                                                                                    | Required           |
+| ------------ | ------------ | ------------------------------------------------------------------------------------------------------------------------------ | ------------------ |
+| `categories` | list of strings | Template must have categories defined.                                                                                          | Yes |
+| `extension`  | object | Template might implement an extension point. One of its properties must be `serviceCode`, which will contain the extension metadata. | No |
+| `apis`   | list of objects | List the APIs required to be added to all Workspaces in the App Builder Project, if they don't exist. Each API object must contain `code` as one of its properties, this is the sdk code of the service. | No |
```

Comment:
This filed appears as `apis` and the example still lists `services`. 


Please also see this proposal for the `install.yaml` schema: https://git.corp.adobe.com/devx-services/aio-lib-console-project-installation/pull/1/files#diff-5f9599c1addcd4716b953264f83abee501e3588d24cf1242f906a0536f6c040a

Reply:
See @slitviachenko's comment below